packages/web/src/builders/build/build.impl.ts

```diff
@@ -102,7 +122,7 @@ export function run(options: WebBuildBuilderOptions, context: BuilderContext) {
             isScriptOptimizeOn
           ),
           // ES5 build for legacy browsers.
-          isScriptOptimizeOn
```

Comment:
Let's keep this condition so differential loading is only needed for prod mode.

Reply:
Added back in https://github.com/nrwl/nx/pull/3319/files#diff-02b88e5e7e44e4228c79e1d106b83f7dR125